src/components/Dropzone/ConfirmRemoveFileModal.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <Modal
+    :visible="visible"
+    :close-button-aria-label="t('dropzone.confirmRemoveFileModal.closeButtonAriaLabel')"
+    @close="$emit('close')"
+  >
+    <div class="px-5 flex flex-col items-center">
+      <TriangleExclamation
+        size="xxl"
+        class="mb-3"
+      />
+      <div
+        class="mb-2 type-xl-600 max-w-[246px] text-center"
+      >
+        {{ title }}
+      </div>
+      <div
+        class="mb-6 type-base-400 max-w-[320px] text-center"
+      >
+        {{ subtext }}
+      </div>
+      <div class="flex w-full justify-center">
+        <LobButton
+          size="small"
+          class="mr-4"
+          @click="$emit('close')"
+        >
+          {{ t('dropzone.confirmRemoveFileModal.goBack') }}
+        </LobButton>
+        <LobButton
+          size="small"
+          variant="secondary"
+          @click="$emit('confirmClicked'); $emit('close')"
+        >
+          {{ confirmButtonText }}
+        </LobButton>
+      </div>
+    </div>
+  </Modal>
+</template>
+
+<script>
+import { Modal } from '@/components';
+import { TriangleExclamation } from '../Icons';
+import { LobButton } from '@/components';
+export default {
+  name: 'ConfirmRemoveFileModal',
+  components: { Modal, TriangleExclamation, LobButton },
+  props: {
+    visible: {
+      type: Boolean,
+      required: true
+    },
+    title: {
+      type: String,
+      default: ''
```

Comment:
should `title` and `subtext` be required? how would the modal look like if these are not passed in?